content/src/Blip.Api.Template/Models/GenericBaseResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace Blip.Api.Template.Models
+{
+    [DataContract]
+    public class GenericBaseResponse
```

Comment:
why use `Generic` and `Base`? It is redundant, specially since this is not using `Generics` (`T`)